evalml/tests/component_tests/test_oversampler.py

```diff
@@ -46,6 +43,8 @@ def test_sampler_selection(
     categorical_columns,
     mock_imbalanced_data_X_y,
 ):
+    from imblearn import over_sampling as im
```

Comment:
Does this mean that if we want to use a non-core dependency in our tests, we now have to import it in every test that we want to use it in? 😮 

I wonder if this will cause the tests to run longer, since we have to import the same modules over and over. Probably not a noticeable difference given our tests but it makes me wonder if there's a way to create a fixture or something the way that we do for our estimators, and then call that instead of importing: `catboost = import_or_raise("catboost", error_msg=cb_error_msg)`